### PR TITLE
2018/07 - PyCon Russia - Machine Learning on Source Code

### PR DESCRIPTION
**Pull request by vmarkovtsev**
Updates: https://github.com/src-d/conferences/issues/49

### CODE REVIEW

File: 2018-pycon-russia/MLonCode.md [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+**Updates**:  https://github.com/src-d/conferences/issues/49
+**Sent on**:  2018/03/20
+**Status**:   submitted
+**Author**:   Vadim Markovtsev
+**Slides**:   TBA
+**Proposal**: N/A
+
+**Abstract**:
+
+Machine Learning on Source Code (MLoSC) is an emerging and exciting domain of research which stands at the sweet spot between deep learning, natural language processing, social science and programming. The list of MLoSC resources - [awesome-machine-learning-on-source-code](https://github.com/src-d/awesome-machine-learning-on-source-code) - has already attracted more than 1,600 watchers on GitHub.
```
Comment (campoy) [inline 2018-pycon-russia/MLonCode.md]: MLonCode

File: 2018-pycon-russia/MLonCode.md [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+
+I will summarize the current research subdomains in MLoSC, such as *identifier and structural embeddings*, *programming language modeling*, *automated programming language evolution* and *source code topic modeling and exploratory search*. I will also describe the open MLoSC toolbox from [source{d}](https://sourced.tech) based on PySpark: [sourced.engine](https://github.com/src-d/engine) and [sourced.ml](https://github.com/src-d/ml). It provides an in-depth set of Python APIs and command line applications which is already being used for various production tasks. Finally, I will show how to apply that toolbox to analyze all world's Python open source code and mine gems.
+
+**Notes**:
+
+About speaker:
```
Comment (campoy) [inline 2018-pycon-russia/MLonCode.md]: Please fix the formatting here.

File: 2018-pycon-russia/MLonCode.md [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+**Status**:   submitted
+**Author**:   Vadim Markovtsev
+**Slides**:   TBA
+**Proposal**: N/A
+
+**Abstract**:
+
+Machine Learning on Source Code (MLoSC) is an emerging and exciting domain of research which stands at the sweet spot between deep learning, natural language processing, social science and programming. The list of MLoSC resources - [awesome-machine-learning-on-source-code](https://github.com/src-d/awesome-machine-learning-on-source-code) - has already attracted more than 1,600 watchers on GitHub.
+
+I will summarize the current research subdomains in MLoSC, such as *identifier and structural embeddings*, *programming language modeling*, *automated programming language evolution* and *source code topic modeling and exploratory search*. I will also describe the open MLoSC toolbox from [source{d}](https://sourced.tech) based on PySpark: [sourced.engine](https://github.com/src-d/engine) and [sourced.ml](https://github.com/src-d/ml). It provides an in-depth set of Python APIs and command line applications which is already being used for various production tasks. Finally, I will show how to apply that toolbox to analyze all world's Python open source code and mine gems.
```
Comment (campoy) [inline 2018-pycon-russia/MLonCode.md]: I thought `gems` was a Ruby term, not Python?

File: 2018-pycon-russia/MLonCode.md (hunk 1 above)
Comment (vmarkovtsev) [reply 2018-pycon-russia/MLonCode.md]: I bet I used the correct hashtag in the actual submission and this is just a copy-paste bug

File: 2018-pycon-russia/MLonCode.md (hunk 3 above)
Comment (vmarkovtsev) [reply 2018-pycon-russia/MLonCode.md]: Hehe. These are the real gems which you mine. Python has wheels and eggs.

File: 2018-pycon-russia/MLonCode.md [hunk 4]
```diff
@@ -0,0 +1,29 @@
...
+
+Machine Learning on Source Code (MLonCode) is an emerging and exciting domain of research which stands at the sweet spot between deep learning, natural language processing, social science and programming. The list of MLonCode resources - [awesome-machine-learning-on-source-code](https://github.com/src-d/awesome-machine-learning-on-source-code) - has already attracted more than 1,600 watchers on GitHub.
+
+I will summarize the current research subdomains in MLonCode, such as *identifier and structural embeddings*, *programming language modeling*, *automated programming language evolution* and *source code topic modeling and exploratory search*. I will also describe the open MLonCode toolbox from [source{d}](https://sourced.tech) based on PySpark: [sourced.engine](https://github.com/src-d/engine) and [sourced.ml](https://github.com/src-d/ml). It provides an in-depth set of Python APIs and command line applications which is already being used for various production tasks. Finally, I will show how to apply that toolbox to analyze all world's Python open source code and mine gems.
+
+**Notes**:
+
+#### About speaker
+
+Lead Engineer, Machine Learning team at https://sourced.tech
```
Comment (campoy) [inline 2018-pycon-russia/MLonCode.md]: Maybe format this is a list? It is not very readable in the current format once the markdown is rendered

**Review by campoy — CHANGES_REQUESTED** [review]

File: 2018-pycon-russia/MLonCode.md (hunk 4 above)
Comment (vmarkovtsev) [reply 2018-pycon-russia/MLonCode.md]: You like to review two times, don't you :)

Comment (vmarkovtsev) [reply 2018-pycon-russia/MLonCode.md]: I copy-pasted this from the actual submission form.

**Comment by campoy** [comment]
I expected you to simply add extra lines, or maybe make it an actual list ...but I'll allow making it a code section I guess 😆

**Review by campoy — APPROVED** [review]

**Pull request merged**